src/hb-directwrite.cc

```diff
@@ -919,7 +924,7 @@ hb_directwrite_shape_experimental_width(hb_font_t          *font,
   unsigned int        num_features,
   float               width)
 {
-  static char *shapers = "directwrite";
+  static char *shapers = (char *) "directwrite";
```

Comment:
This is wrong. You should just declare "static const char *...".

Reply:
Oh, thanks.